course05/meeting00/CMakeLists.txt

```diff
@@ -1,4 +1,15 @@
 cmake_minimum_required(VERSION 3.1)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -g -Wall")
+# Would you normally default CMAKE_EXPORT_COMPILE_COMMANDS to on? And is this the way to do it?
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
My preference is to not enable the generation by default; it is not needed to build the project and if the user wants it, he/she can enable it.